Felienne/hedy
# Adding blur button after program is submitted

**Description**

Adds a blur button that hides the code in the exercise after a program has been submitted 

**Fixes _issue or discussion number_**

Fixes [https://github.com/Felienne/hedy/issues/3250](https://github.com/Felienne/hedy/issues/3250)

**How to test**

* Go to [http://localhost:8080/hedy](http://localhost:8080/hedy)
* Hit the run code button
* After the success message disappears, the blur button will be available
* Click the blur button to ensure the code is blurred out
